lib/client/Client.js

```diff
@@ -536,6 +529,31 @@ export default class Client {
     return this.updateMessage(id, { mailboxIds: mailboxIds });
   }
 
+  _createMessage(message, role, isDraft) {
```

Comment:
Why do not set `isDraft` in the calling method instead of taking a boolean there ?